lineman/app/components/thread_page/comment_form/comment_form.scss

```diff
@@ -4,6 +4,8 @@
 
 .comment-form {
   @include card;
+  /*make room for the "turn off angular" link */
+  margin-bottom: 5em;
```

Comment:
no reason. temporary fix until we design a proper solution for interface
switching

On Fri, Jun 19, 2015 at 11:22 AM James Kiesel <notifications@github.com>
wrote:

> In lineman/app/components/thread_page/comment_form/comment_form.scss
> <https://github.com/loomio/loomio/pull/2255#discussion_r32789205>:
>
> > @@ -4,6 +4,8 @@
> >
> >  .comment-form {
> >    @include card;
> > +  /*make room for the "turn off angular" link */
> > +  margin-bottom: 5em;
>
> Why em?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2255/files#r32789205>.
>